plugins/llvm/llvm_binary.hpp

```diff
@@ -458,28 +458,29 @@ struct objectfile_image : image {
     std::vector<seg::segment> segments_;
     std::vector<sym::symbol> symbols_;
     std::vector<sec::section> sections_;
-    std::unique_ptr<object::Binary> binary_;
+    std::unique_ptr<T, std::default_delete<object::Binary>> binary_;
```

Comment:
There is no  need to specify the default deleter, it is an optional argument, just `std::unique_ptr<T>` will be sufficient